src/net.cpp

```diff
@@ -348,6 +348,11 @@ CNode* CConnman::FindNode(const CService& addr)
     return nullptr;
 }
 
+bool CConnman::AlreadyConnectedToPeer(const CAddress& addr)
```

Comment:
nit: `AlreadyConnectedToAddress()` would be a more accurate name.

Reply:
Agreed, done.